Formula/dvm.rb

```diff
@@ -0,0 +1,35 @@
+class Dvm < Formula
+  desc "Docker Version Manager"
+  homepage "https://github.com/getcarina/dvm"
+  url "https://github.com/getcarina/dvm.git",
```

Comment:
Does this need to be a git checkout or can it use the automatically generated tarball? https://github.com/getcarina/dvm/archive/0.6.2.tar.gz